src/runtime/virtcontainers/README.md

```diff
@@ -14,12 +14,12 @@ Ideally, VM-based container runtime implementations would become translation
 layers from the runtime specification they implement (e.g. the [OCI runtime-spec][oci]
 or the [Kubernetes CRI][cri]) to the `virtcontainers` API.
 
-`virtcontainers` was used as a foundational package for the [Clear Containers][cc] [runtime][cc-runtime] implementation.
+`virtcontainers` was used as a foundational package for the [Kata Containers][kata-containers] [runtime][kata-containers-runtime] implementation, formerly the Clear Containers runtime implementation.
 
 [oci]: https://github.com/opencontainers/runtime-spec
 [cri]: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/container-runtime-interface.md
-[cc]: https://github.com/clearcontainers/
```

Comment:
it is interesting that the clear-containers project is behind a firewall now.
cc @chavafg @jodh-intel  
This section was meant to document its pervious usage, if we cant make the clearcontainers project public facing, perhaps drop this section altogether.

Reply:
This is indeed an interesting one.  I can access it without being connected in the VPN, via a browser on the incognito mode, which makes me think this is not actually behind the VPN.

Reply:
@amshinde @fidencio I can also access it incognito and see "Clearcontainers [DEPRECATED] - Migrate to Kata Containers ", but not otherwise, unless I am on the VPN

Reply:
Intel folks, FWIW I can access `https://github.com/clearcontainers/` without going through the Intel VPN obviously :wink: